webapp/channels/src/utils/license_utils.ts

```diff
@@ -103,3 +104,14 @@ export const licenseSKUWithFirstLetterCapitalized = (license: ClientLicense) =>
     const sku = license.SkuShortName;
     return sku.charAt(0).toUpperCase() + sku.slice(1);
 };
+
+export function isEnterpriseOrCloudOrSKUStarterFree(license: ClientLicense, subscriptionProduct: Product | undefined, isEnterpriseReady: boolean) {
```

Comment:
0/5 but since the license, product, and `isEnterpriseReady` all come from redux, I think we could turn `isEnterpriseOrCloudOrSKUStarterFree` into a full selector that gets all of those from redux.

Also, I'm still trying to understand what we're trying to do by checking for these. Are we trying to determine if we can upsell the user on Enterprise?

Reply:
this checks if we are on a starter licence, if yes then we are checking if this is an enterprise ready app then we are adding an upsel